whitelist/whitelist.go

```diff
@@ -0,0 +1,73 @@
+// Package whitelist provides methods for managing an app's whitelisted IP's.
+package whitelist
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	deis "github.com/deis/controller-sdk-go"
+	"github.com/deis/controller-sdk-go/api"
+)
+
+// List IP's whitelisted for an app.
+func List(c *deis.Client, appID string) (api.Whitelist, error) {
```

Comment:
we can't remove from the app settings as its used for the routable and maintenance also....here the response is just the whitelist and not the full app settings